### PR TITLE
Added Master Slave Reporter

### PR DESCRIPTION
**Pull request by carlos-m159**
The idea was to create a node to aggregate the status of a multiple-lidar setup.

### CODE REVIEW

**Review by aantunes-locus — APPROVED** [review]

File: msg/MasterSlaveStatus.msg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+urg_node/Status master
```
Comment (svwilliams) [inline msg/MasterSlaveStatus.msg]: So.....
Something is going to need to subscribe to this message. And that thing probably doesn't need the entire Hokuyo driver installed. It just needs the message definition. Having the message def inside the lidar driver package is a bit awkward.

Comment (carlos-m159) [reply msg/MasterSlaveStatus.msg]: You are right. Going to create a new pkg

**Review by svwilliams — APPROVED** [review]

**Review by nickovaras — APPROVED** [review]
Code looks good, but the assumption that robots will always use master-slave configuration may not be true in the future.

**Comment by carlos-m159** [comment]
>  the assumption that robots will always use master-slave configuration may not be true in the future.

Stephen mentioned a similar concern so I am going to drop the Master slave message and just use an array of Status

**Pull request merged**